src/services/sessions.js

```diff
@@ -82,6 +84,17 @@ module.exports = class SessionsHelper {
 				})
 			}
 
+			if (!bodyData.mentor_id) {
```

Comment:
@nevil-mathew we should validate if the mentor ids are valid and as per policy, also type and mentor_id are exclusive, meaning type can be pvt/public. mentor_id if given in req body we should take that else use the logged in user's id.